src/ResourceManager/Resources/Commands.Resources/help/Remove-AzureRmManagementGroup.md

```diff
@@ -0,0 +1,149 @@
+---
+external help file: Microsoft.Azure.Commands.ManagementGroups.dll-Help.xml
+Module Name: AzureRM.ManagementGroups
+online version: https://docs.microsoft.com/en-us/powershell/module/remove-azurermmanagementgroup/
+schema: 2.0.0
+---
+
+# Remove-AzureRmManagementGroup
+
+## SYNOPSIS
+Removes a Management Group
+
+## SYNTAX
+
+### GroupOperations (Default)
+```
+Remove-AzureRmManagementGroup [-GroupName] <String> [-PassThru] [-DefaultProfile <IAzureContextContainer>]
+ [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### ManagementGroupObject
+```
+Remove-AzureRmManagementGroup -InputObject <PSManagementGroup> [-DefaultProfile <IAzureContextContainer>]
```

Comment:
Looks like you will need to regenerate this file to get -PassThru in this parameter set.